smallrye-reactive-messaging-provider/src/test/java/io/smallrye/reactive/messaging/inject/EmitterInjectionTest.java

```diff
@@ -581,7 +583,24 @@ public void consume(final String b) {
 
     @Test // Reproduce #511
     public void testWeCanHaveSeveralSubscribers() {
-        EmitterImpl<String> emitter = new EmitterImpl<>("my-channel", "BUFFER", 128, 128);
+        OnOverflow overflow = new OnOverflow() {
```

Comment:
Should we define an Overflow literal?

Reply:
It's in a test, so not sure if it's that important

Reply:
Right, we can keep it like this.